firecloud/fiss.py

```diff
@@ -1226,12 +1228,29 @@ def mop(args):
         # Check output produces a string in Py2, Bytes in Py3, so decode if necessary
         if type(bucket_files) == bytes:
             bucket_files = bucket_files.decode()
+        
+        #List of user submission within this workspace. To ensure deletion of files in the submission directories only
+        user_submission = fapi.list_submissions(args.project, args.workspace)
+        
+        for item in user_submission.json():
+            sub_id = item['submissionId']
+            submission_ids.append(sub_id)
```

Comment:
This is a more "pythonic" way to accomplish this, and removes the need to define `submission_ids` above (which I appreciate for the sake of readability, since python is not scoped within a function):
```python
submission_ids = set(item['submissionId'] for item in user_submission.json())
```